desktop/mac/release-notes.md

```diff
@@ -17,14 +17,43 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-## Docker Desktop 4.0.1
-2021-09-13
+## Docker Desktop 4.1.0
+2021-09-30
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### New
+
+- **Preferences**
+  - New update section to manage updates 
```

Comment:
```suggestion
  - **Software Updates**: The Settings tab now includes a new section to help you manage Docker Desktop updates. The **Software Updates** section notifies you whenever there's a new update and allows you to download the update or view information on what's included in the newer version. For more information, see (../#software-updates).
```

Reply:
I put `[Software Updates](../#software-updates)` instead of `(../#software-updates)`.
On the contrary, in the preview of this page, I couldn't click on this link. Would it be normal? The same for `Compose V2` and `Volume Management`

Reply:
@trungutt The links to Software updates and Volume Management work fine. I've added a suggestion to fix the Compose V2 URL.